include/blogWithComments_functions.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+$Error = array();
+
+
+function RepopulateComments($info){
+		global $Error;
+	if(isset($_REQUEST['submitComment']) && (sizeof($Error)>0) && isset($_REQUEST[$info])){
+		echo "<input type='text' name=$info value = $_REQUEST[$info] >";
+	}
+	else {
+		echo "<input type='text' name=$info>";
+	}
+}
+
+
+function SubmitComment($Name, $Email, $Comment){
+		$result = dbQuery("
+			INSERT INTO BlogComments(Name, Email, Comment)
+			VALUES('$Name', '$Email', '$Comment')
+		")->fetch();
+}
+
+
+if(isset($_REQUEST['submitComment'])){
+	Validate();
+	if((sizeof($Error)) == 0){
+		SubmitComment(
+				$_REQUEST['NameField'],
+				$_REQUEST['EmailField'],
+				$_REQUEST['CommentField']
+		);
+	}
+	if(sizeof($Error) > 0){
+		echo "<p class='ErrorMessage'>In order to submit a comment, please fill out all the fields!</p>";
+	}
+}
+
+
+function Validate(){
```

Comment:
This is a pretty generic name for what looks like a function that validates a specific form.